src/commonTest/kotlin/au/id/micolous/metrodroid/test/ClipperTest.kt

```diff
@@ -105,6 +106,17 @@ class ClipperTest : BaseInstrumentedTest() {
         assertNear(-121.89918, trips[0].endStation!!.longitude!!.toDouble(), 0.00001)
     }
 
+    @Test
+    fun testVehicleNumbers() {
+        assertEquals(null, ClipperTrip("0000000000000000000000000000000000000000000000000000000000000000").vehicleID)
```

Comment:
All of these need to be `ClipperTrip(ImmutableByteArray.fromHex(...` instead to fix the build failure: https://travis-ci.org/metrodroid/metrodroid/jobs/625558147#L438

Reply:
Fixed!